src/CalendarView.vue

```diff
@@ -537,6 +537,10 @@ header are in the CalendarViewHeader component.
 	background-color: #f7f7f7;
 	border-width: 1px;
 }
+.cv-event:hover {
+	cursor: pointer;
+	width:fit-content !important;
+}
```

Comment:
I suggest adding z-index: 1 here, too, else if you have events on adjacent days the title can get obscured

Reply:
yes, you're right, I try to add z-index, it seems better .
thank you for your advice.

Reply:
As currently proposed, this breaks when the long title is on the last day of the week -- it doesn't show the longer title, and can create flashes or impact the metrics of the overall calendar (at least on Chrome OSX).

The component does show the longer title, but it does so using the "title" attribute and normal browser semantics, which is limited, so I can see why there would be a benefit of adjusting the DOM instead.

I've just sent a commit that, rather than expanding the span of the event, keeps whatever span it has but wraps the content and ensures (via z-index) that it appears on top of other events on the same day. This works well even on a crowded day, since the weeks can be scrolled, and requires only one small addition to the stylesheet.

Reply:
adding *white-space: normal*  is a great solution of "title won't show if event on the last day of the week",thank you .

then, I think *min-width* should be leave, because when the title's content  is much less than the *day grid*, and event background-color is different from day background-color, it maybe seems strange.

finally, make it optional can be great, sometimes users just want to see entire title on calendar page.

Reply:
Interesting! I haven't seen any situations where the event width ends up *smaller* than the provided space, even when the title is short and the event is long.

Are you seeing that in a particular browser or with certain event contents or themes?

If we need to move to providing both `min-width` and `width`, I'd be for going ahead and swapping out `width` with `max-width`, just in case.